JellyfishClientDemo/JellyfishClientDemo/ContentViewController.swift

```diff
@@ -6,10 +6,15 @@
 //
 
 import Foundation
-import JellyfishClientSdk
 import MembraneRTC
 import UIKit
 
+import struct JellyfishClientSdk.Config
+import protocol JellyfishClientSdk.JellyfishClientListener
+import class JellyfishClientSdk.JellyfishClientSdk
+import struct JellyfishClientSdk.Peer
+import class JellyfishClientSdk.TrackContext
```

Comment:
Afaik if you want to import one class (TrackContext here) specifically from one library every other thing has to be also included  line by line...